sdk/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Can we rely on the Gradle files in the root folder?

Reply:
same for gradle-wrapper.jar

Reply:
This is auto-generated, my laptop and computer both have problems when trying to use gradle files in the project, so it always takes whatever the root gradle has